integration/mediation-tests/tests-mediator-1/src/test/java/org/wso2/carbon/esb/mediator/test/conditionalRouter/ConditionalRouterIntegrationTest.java

```diff
@@ -318,6 +313,15 @@ public void conditionalRouterMediatorWithEqualHeaderTest() throws Exception {
 
     }
 
+    private Callable<Boolean> dynamicSequenceExists (final String[] seqArr, final String strToFind) {
```

Comment:
Add a method description.

Reply:
Fixed in https://github.com/wso2/product-ei/pull/3665